scenarioo-client/app/step/step.html

```diff
@@ -144,26 +144,35 @@
                     </div>
                 </div>
                 <div id="sc-step-comparison-side-by-side-view" ng-if="comparisonViewOptions[displaySideBySideViewKey]">
-                    <div class="col-lg-6">
-                        {{baseBuild.aliasName}}
-                        <div class="sc-base-screenshot clear" ng-if="!comparisonViewOptions[displayDiffScreenshotKey]">
-                            <sc-screenshot-title build="baseBuild" name="Current" alias="{{baseBuildName}}" highlights="false"></sc-screenshot-title>
-                            <sc-annotated-screenshot screen-shot-url="screenShotUrl" visibility-toggle="showScreenAnnotations"></sc-annotated-screenshot>
+                    <div class="row">
```

Comment:
having two rows layout here seems risky to me ... for small screens it may mean that there will be the headers first and then both screenshots. No one will understand that.